### ISSUE TITLE
SqlGeography support for FULLGLOBE?

### ISSUE THREAD

**Issue by petertirrell** [dotMorten/Microsoft.SqlServer.Types#41]
Are there plans to support creating a `SqlGeography` of "FULLGLOBE"?

A sample test method
```
[TestMethod]
public void FullGlobe() {
    var wkt = "FULLGLOBE";
    var value = SqlGeography.STGeomFromText(new SqlChars(new SqlString(wkt)), 4326);
    Assert.IsNotNull(value);
}
```

this passes in the RefTests but fails with an "invalid well-known text" when using this library.

**Comment by dotMorten** [comment]
Thank you for the nice simple test! That makes it easy to verify once feature is complete (and yes I take PRs).

**Comment by dotMorten** [comment]
Fixed in PR #42
